src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -226,6 +228,27 @@ public override void Dispose()
             _node.WorkspaceViewModel.PropertyChanged -= Workspace_PropertyChanged;
         }
 
+        internal void PlaceNodeAutocompleteWindow(object sender, EventArgs e)
```

Comment:
@aparajit-pratap Would you add some comments for this function? and maybe in the comments also mention how to adjust the offset in the future